microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -176,69 +199,68 @@ The DOM tree is not the HTML source tagging.
 In short, the **DOM Tree** `!==` HTML.
 -->
 
-### Change HTML from the Sources tool
+###### Change HTML from the Sources tool
 
-If you want to save a change to the HTML of the webpage, use the **Sources** tool.
+If you want to save a change to the webpage HTML, use the **Sources** tool.
```

Comment:
```suggestion
If you want to save a change to the webpage HTML, use the **Sources** tool with a workspace defined, rather than using the **Elements** tool.
```